TELCODOCS-903-RN: Release note for Graceful Node Shutdown (#55869)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-nodes"]
=== Nodes

> ==== Graceful Node Shutdown

Reviewer: ```suggestion
[id="ocp-4-13-graceful-node-shutdown"]
==== Graceful Node Shutdown
```

Graceful Node Shutdown (GNS) enables kubelet to delay forcible eviction of pods in the event of a node shutdown.

> This is important for users who want to ensure that workloads that execute critical operations cannot be interrupted.

Reviewer: ```suggestion
This ensures that workloads that execute critical operations are not interrupted.
```

> For example, if an operator that flashes network card firmware is shut in a disorderly fashion, hardware can be damaged.

Reviewer: ```suggestion
For example, if an operator that flashes network card firmware is shut down in a disorderly fashion, hardware can be damaged.
```

> For example, if an operator that flashes network card firmware is shut in a disorderly fashion, hardware can be damaged.

Reviewer: Suggested rewrite for clarity: Delete this example and add it below alongside the description of the function.

GNS gives admins the ability to specify a termination grace period for protected workloads.

> This ensures time for the pod to shut down in an orderly manner, including operations that are monitoring the activity of sensitive workloads.

Reviewer: ```suggestion
This ensures that the pod shuts down in an orderly manner, including allowing operations that are monitoring the activity of sensitive workloads to finish.
```
